data/slash/slash.proto

```diff
@@ -9,17 +9,17 @@ import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 import "github.com/ElrondNetwork/elrond-go-core/data/block/blockV2.proto";
 
 enum SlashingType {
-  None = 0;
+  None             = 0;
   MultipleProposal = 1;
-  MultipleSigning = 2;
+  MultipleSigning  = 2;
 }
 
 enum ThreatLevel {
-  Low = 0;
+  Low    = 0;
```

Comment:
Could there be also a None here like on Type?

Reply:
Yes, it should totally be one.

Reply:
Actually, renamed it to **Zero**

Reply:
ok